tests_integration/__tests__/__snapshots__/early-exit.js.snap

```diff
@@ -1,5 +1,72 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`show detailed usage with --help config-precedence 1`] = `
+"Config Precedence (--config-precedence <cli-override|file-override|prefer-file>)
+
+  Define in which order config files and CLI options should be evaluated.
+  Defaults to cli-override.
+
+Valid options:
+
+  cli-override    CLI options take precedence over config file
+  file-override   Config file take precedence over CLI options
+  prefer-file     If a config file is found will evaluate it and ignore other CLI options.
+                  If no config file is found CLI options will evaluate as normal.
+
+Default: cli-override
+"
+`;
+
+exports[`show detailed usage with --help cursor-offset 1`] = `
+"Cursor Offset (--cursor-offset <int>)
+
+  Print (to stderr) where a cursor at the given position would move to after formatting.
+  This option cannot be used with --range-start and --range-end.
+
+Default: -1
+"
+`;
+
+exports[`show detailed usage with --help no-option 1`] = `
+"No Semi (--no-semi)
+
+  Do not print semicolons, except at the beginning of lines which may need them.
+"
+`;
+
+exports[`show detailed usage with --help parser 1`] = `
+"Parser (--parser <flow|babylon|typescript|postcss|json|graphql>)
+
+  Which parser to use. Defaults to babylon.
+
+Valid options:
+
+  flow         undocumented
+  babylon      undocumented
+  typescript   undocumented
+  postcss      undocumented
+  json         undocumented
+  graphql      undocumented
+
+Default: babylon
+"
+`;
+
+exports[`show detailed usage with --help trailing-comma 1`] = `
+"Trailing Comma (--trailing-comma <none|es5|all>)
+
+  Print trailing commas wherever possible when multi-line. Defaults to none.
+
+Valid options:
+
+  none   No trailing commas.
+  es5    Trailing commas where valid in ES5 (objects, arrays, etc.)
+  all    Trailing commas wherever possible (including function arguments).
+
+Default: none
```

Comment:
It's a bit unfortunate that it says "Defaults to none." above and then we repeat "Default: none" again here. Is there anything we can do about that?